test/test-helpers.js

```diff
@@ -0,0 +1,175 @@
+const { spawnSync } = require('child_process')
+const os = require('os')
+
+const isWin = (os.platform() === 'win32')
+const pwd = process.cwd()
+const whiteSpaceReg = /[\\\s]/g
+
+/**
+ * Function: runc8
+ *
+ * @param {Array|String} args: an array of arguments or a string argument
+ * to pass to child spawned process.
+ * @param {Object} options: an Object with the following key-values
+ *  {Object} spawnSyncOptions: The options object found here. http://tinyurl.com/66ztyx9b
+ *  {String} expectedOutput: either 'text' or 'json' - default 'json'
+ *  {Boolean} stripWhiteSpace: Should the output returned be stripped of all whitespace.
+ *    - default false
+ *  {Boolean} removeBannerDivider: Should remove the divider line below the config
+ *    report banner - default false
+ * @returns {String} out: a string representing the stdout
+ *   of the child process
+ *
+ *
+ */
+const runc8 = (args, options) => {
+  const nodePath = process.execPath
+  const c8Path = require.resolve('../bin/c8')
+  const argsv = (typeof args === 'string')
+    ? [c8Path, args]
+    : [c8Path, ...args]
+
+  return runSpawn(nodePath, argsv, options)
+}
+
+/**
+ * Function: runSpawn
+ *
+ * @param {String} cmd: A string representing the prompt command
```

Comment:
This seems like quite a bit of test helper logic being introduced for the size of the feature.

Could we get away with simply using a regex assertion and testing against some of the expected output?